### PR TITLE
Upgrade the cookie policy  to 3.0.3

### LINKED ISSUE #42: Add cookie policy and update GTM
is this the actual repo?

## On the site
in package.json
```
"@canonical/cookie-policy": "3.0.2",
```

in templates/partial/_footer.html
```
<a href="" class="js-revoke-cookie-manager">Manage your tracker settings</a>
```

### add the js (depends on site)
static/js/base.js
```
import { cookiePolicy } from "@canonical/cookie-policy";
cookiePolicy();
```
or
```
<script src="/static/js/modules/cookie-policy/cookie-policy.js"></script>
<script src="/static/js/modules/global-nav/global-nav.js"></script>
<script>
  canonicalGlobalNav.createNav({
    maxWidth: '72rem',
  });
  cpNs.cookiePolicy();
</script>
```

### add to scss
static/sass/styles.scss
```
// import cookie policy
@import "@canonical/cookie-policy/build/css/cookie-policy";
```

## On the GTM

1. add variable - `_cookies_accepted`
2. add trigger - `All pages && cookie (all or functionality)` - page view regex match `(all|functionality)`
3.add trigger -  `All pages && cookie (all or performance)` - regex match `(all|performance)`
4. add these to the triggers based on [this sheet](https://docs.google.com/spreadsheets/d/1PQEmbJPEG-76FLtLVOO271615V5aor66UhxfomZF-lo/edit#gid=0}
5. tracker `track pageview for cookie acceptance` for pageview
6. trigger - custom `track pageview event` on event `pageview`
#### performance
* ga
* crazyegg
* usabilla
* vimeo	
* google optimise

#### functionality
* linked in	
* livechat	
* facebook	
* twitter	
* bombora	
* google adwords	
* Marketo Munchkin

### PR DESCRIPTION
**Pull request by anthonydillon**
## QA
- Run `./run clean` and `./run`
- Go to http://0.0.0.0:8031/
- Check the cookie policy is working as expected.

Fixes https://github.com/canonical-web-and-design/landscape-brochure.internal/issues/42

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     ]
   },
   "dependencies": {
-    "@canonical/cookie-policy": "2.0.3",
+    "@canonical/cookie-policy": "3.0.3",
```
Comment (pmahnke) [inline package.json]: 3.04

**Comment by anthonydillon** [comment]
@pmahnke done

**Comment by pmahnke** [comment]
@anthonydillon is this even live? it seems private so circleci is having issues... and I am not sure how it gets added to the live site?

**Comment by anthonydillon** [comment]
It's not, I have been in talks with landscape but they are keen on it.

**Review by pmahnke — APPROVED** [review]
+1

**Pull request merged**